instrumentation/aws_sdk/gemfiles/aws_sdk_3.0.gemfile

```diff
@@ -10,6 +10,7 @@ group :test do
   gem "opentelemetry-common", path: "../../../common"
   gem "opentelemetry-sdk", path: "../../../sdk"
   gem "opentelemetry-semantic_conventions", path: "../../../semantic_conventions"
+  gem "webrick", "~> 1.7"
```

Comment:
Why add webrick now?

Reply:
Regenerating the appraisal files brought in `webrick` because if you take a look at the Gemfile you will see it is an explicit dependency:

https://github.com/open-telemetry/opentelemetry-ruby/blame/main/instrumentation/aws_sdk/Gemfile#L18

`webrick` was added to the main Gemfile _after_ the appraisals were already generated but not updated at the time to include it.

This is the commit that introduced the appraisals: 2021-11-04
https://github.com/open-telemetry/opentelemetry-ruby/pull/1014/commits/32335be10b61f82d65f208476ecbc67f098b21c9

This is commit is were `webrick` was added but the appraisals were not updated along with it: 2021-11-07
https://github.com/open-telemetry/opentelemetry-ruby/pull/1014/commits/d57cd30a25372179fbb5933b485764f6723c69fe